app/gemini-bu-rot/app.toml

```diff
@@ -118,24 +118,29 @@ task-slots = ["syscon_driver"]
 [tasks.spi0_driver]
 name = "drv-lpc55-spi-server"
 priority = 4
-max-sizes = {flash = 16384, ram = 2048}
+max-sizes = {flash = 65536, ram = 32768}
```

Comment:
Running `cargo xtask sizes` shows this task is actually using 13K of flash and 19K of RAM.  Any idea why the RAM usage is so high?

Reply:
Sprockets gets linked into this task. The flash limit can be reduced.

Reply:
I'll leave further memory footprint work for a future PR.